vivisect/analysis/generic/codeblocks.py

```diff
@@ -16,6 +16,7 @@
 def analyzeFunction(vw, funcva):
     blocks = {}
     done = {}
+    mnem = {}
```

Comment:
if you make this mnem dict more like:

`mnem = collections.defaultdict(int)`

then the lines later can avoid the extra conditional ( and avoid an extra dict hash lookup ) via

`mnem[op.mnem] += 1`

which would be great for perf